File: 91-aws/02-rest-api-mysql/src/main/java/com/in28minutes/rest/webservices/restfulwebservices/versioning/PersonV2.java
package com.in28minutes.rest.webservices.restfulwebservices.versioning;

public class PersonV2 {
	private Name name;

	public PersonV2(Name name) {
		super();
		this.name = name;
	}

	public Name getName() {
		return name;
	}

	@Override
	public String toString() {
		return "PersonV2 [name=" + name + "]";
	}

}

File: 12-rest-api-kotlin/src/main/kotlin/com/in28minutes/rest/webservices/restfulwebservices/versioning/PersonV2.kt
package com.in28minutes.rest.webservices.restfulwebservices.versioning

data class PersonV2(val name: Name)

// OR

//class PersonV2(private val name: Name) {
//
//    // Explicit getter for serialization
//    fun getName(): Name = name
//
//    override fun toString(): String {
//        return "PersonV2 [name=$name]"
//    }
//}

